changelogs/current.yaml

```diff
@@ -206,6 +206,12 @@ removed_config_or_runtime:
 - area: runtime
   change: |
     removed ``envoy.restart_features.no_runtime_singleton`` and replaced with ``envoy.restart_features.remove_runtime_singleton``.
+- area: udp listener
```

Comment:
added support for command operators in :ref:`TunnelingConfig hostname 

doesn't seem like a backwards incompatible change